### ISSUE TITLE
Spriggit's UI gives no notification when the jobs are done.

### ISSUE THREAD

**Issue by gir489returns** [Mutagen-Modding/Spriggit#85]
UX101 is to always keep your user informed of what is going on with the program. Silently doing nothing is the worst thing possible. At the end of a sync job, no notification is displayed or anything in the UI changed to let the user know if the job succeeded. If it fails, a large (sometimes mostly useless) error box is displayed. This can become troublesome when dealing with potentially large ESPs as for some reason, the CK keeps trying to pack worldspace changes into my ESP that I didn't even load or touch, inflating my ESP from 700KBs to over 5MBs, causing the Spriggit job to go from 3 seconds to about a whole minute.

I only figure this out later when doing code reviews that some YAMLs are missing do I then go investigating as to the cause, which is usually overzealous reverting of YAMLs covered in another ticket, but sometimes it's caused by this WoldSpace inflating nonsense.

TL;DR: A simple messagebox that the job is complete would be nice.

**Comment by Noggog** [comment]
Yup!  Most definitely.  The UI is definitely in a half baked space.   I hope to give it the final passover before pushing 1.0
